changelog/0.0.9.md

```diff
@@ -11,7 +11,6 @@
 ## New features
 
 * Convenience support for AuraDS connections by configuring the Python driver with recommended non-default configuration.
-* Added support for GDS 1.x `graph.create` syntax.
```

Comment:
We should not remove it from here, but instead declare in `1.1.1.md` that we have removed it :)

Reply:
Done